libraries/node-core-library/src/Terminal/test/__snapshots__/Terminal.test.ts.snap

```diff
@@ -0,0 +1,799 @@
+
+exports[`01 color enabled 02 verbose turned on 01 writeVerbose 05 writes a messages with colors interspersed with non-colored messages 1`] = `"message 1[32mmessage 2[39mmessage 3[31mmessage 4[39m"`;
+
+exports[`01 color enabled 02 verbose turned on 01 writeVerbose 05 writes a messages with colors interspersed with non-colored messages 2`] = `""`;
+
+exports[`01 color enabled 02 verbose turned on 01 writeVerbose 05 writes a messages with colors interspersed with non-colored messages 3`] = `""`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 01 writes a single message 1`] = `
+"test message
+"
+`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 01 writes a single message 2`] = `""`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 01 writes a single message 3`] = `""`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 02 writes multiple messages 1`] = `
+"message 1message 2
+"
+`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 02 writes multiple messages 2`] = `""`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 02 writes multiple messages 3`] = `""`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 03 writes a message with colors 1`] = `
+"[32mmessage 1[39m
+"
+`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 03 writes a message with colors 2`] = `""`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 03 writes a message with colors 3`] = `""`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 04 writes a multiple messages with colors 1`] = `
+"[32mmessage 1[39m[31mmessage 2[39m
+"
+`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 04 writes a multiple messages with colors 2`] = `""`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 04 writes a multiple messages with colors 3`] = `""`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 05 writes a messages with colors interspersed with non-colored messages 1`] = `
+"message 1[32mmessage 2[39mmessage 3[31mmessage 4[39m
+"
+`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 05 writes a messages with colors interspersed with non-colored messages 2`] = `""`;
+
+exports[`01 color enabled 02 verbose turned on 02 writeVerboseLine 05 writes a messages with colors interspersed with non-colored messages 3`] = `""`;
+
+exports[`01 color enabled 04 toggling verbose on and off 1`] = `
+"SHOULD BE PRINTED
+SHOULD BE PRINTED
+SHOULD BE PRINTED
+SHOULD BE PRINTED
+"
+`;
+
+exports[`01 color enabled 04 toggling verbose on and off 2`] = `""`;
+
+exports[`01 color enabled 04 toggling verbose on and off 3`] = `""`;
+
+exports[`01 color enabled 05 writes to multiple streams 1`] = `
+"message 1message 2[32mmessage 1[39mtest message
+test messagemessage 1message 2
+[32mmessage 1[39m
```

Comment:
>[32mm [](start = 0, length = 6)

To make the snapshot more readable, maybe you could search+replace 0x1b with something like [b]?  My text editor renders it as nothing at all.